python/plasma/plasma.py

```diff
@@ -129,6 +129,10 @@ def create(self, object_id, size, metadata=None):
       metadata (buffer): An optional buffer encoding whatever metadata the user
         wishes to encode.
 
+    Returns:
+      A buffer for the object created. If the object wasn't able to be created
+        because it already exists locally, returns None.
+
     Raises:
       plasma_object_exists_error: This exception is raised if the object could
```

Comment:
Oh, yes we're still using the exception. I think this comment is left over from when I was rebasing.